static/js/giphy.js

```diff
@@ -103,7 +103,7 @@ export function initialize() {
     $("body").on("keydown", ".giphy-gif", ui_util.convert_enter_to_click);
     $("body").on("keydown", "#compose_giphy_logo", ui_util.convert_enter_to_click);
 
-    $("#compose_giphy_logo").on("click", (e) => {
+    $("body").on("click", "#compose_giphy_logo", (e) => {
```

Comment:
We need to add another commit changing this from an ID to a class -- duplicate IDs in a document are a violation of the HTML spec, and with this model we'll have a duplicate if a `message_edit` form is open.

Reply:
Done